### ISSUE TITLE
Show play status and Now playing in HTML Title tag in mopidy musicbox web client

### ISSUE THREAD

**Issue by MurzNN** [pimusicbox/mopidy-musicbox-webclient#243]
At now page title contains only "Mopidy on hostname" text, so this is not so useful. Will be good to dynamically update title and show in them:
- current play status: play / pause / stop / buffering (via utf-8 symbol, example how to do this you can look at the Mopify and Iris web client)
- title of current playing song (from Now playing, example in Iris web client)
- title of current menuitem (Playlists, Streams, etc)

Suggestion for title template: [play status symbol] [now playing] - [current menuitem] | Mopidy on [hostname]

**Comment by MurzNN** [comment]
Another good way to display play status will be page favicon - you also can dynamically update via JS it without page reload.

**Comment by jcass77** [comment]
This sounds like a cool idea, however having looked at more closely I think it poses a few challenges:

- there doesn't seem to be good cross-browser support for dynamically changing and refreshing the favicon without also forcing a page reload (at least not that I could find). Even if we managed to find a workaround for this, we'll then have to generate and distribute static .ico files of our own if we want to display the same font-awesome icons as used elsewhere in MMW. This adds a bit of a maintenance burden going forward (as we'll have to keep this in sync with each font-awesome release) for a fairly minor UI tweak.
- If resorting to plain text and unicode characters, then there also [aren't any consistent unicode characters for 'pause'](http://www.unicode.org/mail-arch/unicode-ml/y2005-m08/0367.html) to use along with [&#9654; - 'black right pointing arrow'](http://www.fileformat.info/info/unicode/char/25B6/index.htm). All of the alternatives that I tried looked weird.
- (probably most importantly) changing the title to include details of the currently playing song pollutes the browser history: the browser history will show ``[play status symbol] [now playing] - [current menuitem] | Mopidy on [hostname]``, while the URL for that history entry only contains the hash to navigate to ``[current menu item]``. It might be surprising to users if they click on a history entry like this expecting to hear the same song again while a different one is currently playing (or worse still, paused).

Having said all of that, I think it would be useful to at least include ``[current menu item]``, as this will make it easier to select a specific menu items again when going back in the browser history.
